api/setup.md

```diff
@@ -3,6 +3,7 @@ layout: default
 title: Setting up the API
 parent: API
 nav_order: 1
+published: false
```

Comment:
Most of this documentation relates to API key setup, which is not needed at this time.